style.css

```diff
@@ -148,6 +148,434 @@ body {
+.bottom-section__title,
+.bottom-section__subtitle {
+  margin: 0;
+  margin-top: 5px;
+}
+
+.bottom-section__description {
+  margin-bottom: 22px;
+  opacity: .5;
+}
+
+.bottom-section__btn_other {
+  position: relative;
+  width: 138px;
+  padding-bottom: 30px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: normal;
+  cursor: pointer;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #000;
+}
+
+.bottom-section__arrow {
+  cursor: pointer;
+  position: relative;
+}
+
+.bottom-section__arrow div {
+  position: relative;
+  width: 108px;
+  height: 2px;
+  background-color: #000;
+  display: block;
+}
+
+.bottom-section__arrow div::after {
+  content: '';
+  position: absolute;
+  width: 7px;
+  height: 1px;
+  top: -3px;
+  right: 0;
+  background-color: #000;
+  transform: rotate(45deg);
+}
+
+.bottom-section__arrow div::before {
+  content: '';
+  position: absolute;
+  width: 7px;
+  height: 1px;
+  top: 4px;
+  right: 0;
+  background-color: #000;
+  box-shadow: 0 3px 5px rgba(0, 0, 0, .2);
+  transform: rotate(-45deg);
+}
+
+.bottom-section__btn_basket {
+  width: 203px;
+  height: 30px;
+  text-transform: uppercase;
+  border: 1.5px solid #000000;
+  text-align: center;
+  width: 256px;
+  padding-top: 9px;
+  padding-bottom: 26px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+  text-align: center;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+  cursor: pointer;
+}
+
+.footer {
+  display: none;
+}
+
+@media (min-width: 1400px) {
+  .middle-section__btn_other {
+    right: 78px;
+    bottom: 121px;
+    text-decoration: none;
+  }
+
+  .bottom-section__image {
+    margin-left: 48px;
+    margin-right: 36%;
+  }
+
+  .bottom-section__img {
+    margin-bottom: 141px;
+  }
+
+  .bottom-section__block {
+    top: 28%;
+    padding-left: 43px;
+    padding-right: 57px;
+    margin-right: 7%;
+    margin-left: 57%;
+  }
+
+  .bottom-section__title {
+    margin-top: 11px;
+    margin-bottom: 10px;
+  }
+
+  .bottom-section__subtitle {
+    margin-bottom: 18px;
+  }
+
+  .bottom-section__description {
+    margin-bottom: 55px;
+    margin-left: 4px;
+    margin-right: 10px;
+  }
+
+  .bottom-section__btn_other {
+    right: 10%;
+    bottom: 27%;
+  }
+}
+
+@media (min-width: 1300px) {
+  .middle-section {
+    padding-right: 73px;
+    padding-bottom: 137px;
+    margin: 0;
+  }
+
+  .middle-section__image {
+    margin-top: 128px;
+    margin-right: 42px;
+    margin-left: 100px;
+  }
+
+  .middle-section__btn_other {
+    right: 6%;
+    text-decoration: none;
+  }
+}
+
+@media (min-width: 1250px) {
+  .bottom-section {
+    background: -webkit-gradient(linear, left top, right top, color-stop(27%, #EBE3D8), color-stop(27%, #000), color-stop(32%, #000), color-stop(32%, #EBE3D8));
+    background: -webkit-linear-gradient(left, #EBE3D8 27%, #000 27%, #000 32%, #EBE3D8 32%);
+    background: -o-linear-gradient(left, #EBE3D8 27%, #000 27%, #000 32%, #EBE3D8 32%);
```

Comment:
Codacy found an issue: [Expected "#EBE3D8" to be "#ebe3d8" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/locoalin/pullRequest?prid=5835770)